### PR TITLE
Add vcpkg installation instructions

### PR DESCRIPTION
**Pull request by JackBoosY**
kcp is available as a port in vcpkg, a C++ library manager that simplifies installation for kcp and other project dependencies. Documenting the install process here will help users get started by providing a single set of commands to build kcp, ready to be included in their projects.

We also test whether our library ports build in various configurations (dynamic, static) on various platforms (OSX, Linux, Windows: x86, x64, UWP, ARM) to keep a wide coverage for users.

I'm a maintainer for vcpkg, and here is [what the port script looks like](https://github.com/microsoft/vcpkg/blob/master/ports/kcp/portfile.cmake). We try to keep the library maintained as close as possible to the original library.

If you have any questions, you can reply to me in Chinese : )

### CODE REVIEW

**Comment by skywind3000** [comment]
thanks

File: README.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ TCP是为流量设计的（每秒内可以传输多少KB的数据），讲究的
...
+
+您可以使用[vcpkg](https://github.com/Microsoft/vcpkg)库管理器下载并安装kcp:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install kcp
+
+vcpkg中的kcp库由Microsoft团队成员和社区贡献者保持最新状态。如果版本过时，请在vcpkg存储库上[创建问题或请求请求](https://github.com/Microsoft/vcpkg)。
```
Comment (JackBoosY) [inline README.md]: 抱歉，我好像在这里翻译错了。
`创建问题或请求请求`应当改为`创建issue或提出PR`。

Comment (JackBoosY) [reply README.md]: 需要我重新提一个PR吗？

Comment (skywind3000) [reply README.md]: 发吧

**Pull request merged**